packages/codegen-ui-react/lib/helpers/index.ts

```diff
@@ -14,3 +32,16 @@
   limitations under the License.
  */
 export const lowerCaseFirst = (input: string) => input.charAt(0).toLowerCase() + input.slice(1);
+
+export const addModelAlias = (componentTypes: string[], importCollection: ImportCollection, model: string) => {
```

Comment:
nit: the rest of our code shouldn't need to be aware of the fact that the model name is potentially aliased. This's bit of collision detection/resolution is being abstracted away and all the rest of the app needs to know is that this is the way to add a model to the import collection

Reply:
In other words, the `importCollection` itself should be able to handle collisions and that the helper function is not needed?

Reply:
I was originally commenting about the naming of the helper, but you took it to the next logical conclusion and I agree that a helper shouldn't be needed.